lib/Elastica/Transport/AbstractTransport.php

```diff
@@ -99,14 +99,13 @@ public static function create($transport, Connection $connection, array $params
             } else {
                 $transport = ucfirst($transport);
             }
-
-            $className = 'Elastica\\Transport\\'.$transport;
-
-            if (!class_exists($className)) {
-                throw new InvalidException('Invalid transport');
+            $classNames = ["Elastica\\Transport\\$transport", $transport];
```

Comment:
Maybe we could check whether namespace separator (\) exists or not? It means it could be FQCN or built-in transport of Elastica right?